src/App.tsx

```diff
@@ -1,35 +1,26 @@
 import React, { useState } from "react";
-import Welcome from "./components/welcome";
+import Welcome from "./pages/Welcome/Welcome";
+import Matches from "./pages/Matches/Matches";
 import firebase, { provider } from "./firebase";
+import {
+  HashRouter as Router,
+  Route,
+  Link,
+  Switch,
+  useHistory,
+} from "react-router-dom";
 
 const App: React.FC = () => {
   const [user, setUser] = useState<firebase.User | undefined>();
 
-  const signIn = async () => {
-    await firebase
-      .auth()
-      .signInWithPopup(provider)
-      .then((result) => {
-        result.user && setUser(result.user);
-      })
-      .catch((error) => {
-        console.log(error);
-      });
-  };
-
-  const signOut = async () => {
-    await firebase
-      .auth()
-      .signOut()
-      .then(() => {
-        setUser(undefined);
-      })
-      .catch((error) => {
-        console.log(error);
-      });
-  };
-
-  return <Welcome signIn={signIn} />;
+  return (
+    <Router>
+      <Switch>
+        <Route exact path="/" component={Welcome} />
+        <Route exact path="/matches" component={Matches} />
```

Comment:
I noticed the url path goes to /#/matches rather than /matches? And if I try to navigate directly to matches it sends me back to the login page (not sure if this is intended functionality or not)

Reply:
also fixed in the second PR now